.gitignore

```diff
@@ -45,3 +45,10 @@ $RECYCLE.BIN/
 Network Trash Folder
 Temporary Items
 .apdisk
+
```

Comment:
Are changes to the .gitignore really necessary?

Reply:
I think so. If node_modules will be not ignored every git will see every dependency as new file.

Reply:
Just thought about this, these rules could be moved to separate .gitignore in _tools subdirectory.